lib/environment_base.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/**
+ * Base class for environments in projects that use the test
+ * runner
+ */
+class EnvironmentBase {
```

Comment:
Can have the `module.exports` up here?

```js
module.exports = class EnvironmentBase {
  ...
};
```

Reply:
frankly I prefer `module.exports` to always be in a well-found location based on convention (e.g. it's _always_ at the bottom of our files generally in these projects). Maybe we should come to consensus and add this to our development guide?

Reply:
Ok, that is fine for now.